Tools/CMake/AMReXParallelBackends.cmake

```diff
@@ -219,4 +229,13 @@ if (AMReX_HIP)
       PUBLIC
       $<$<COMPILE_LANGUAGE:CXX>:-m64 --amdgpu-target=${AMReX_AMD_ARCH}> )
```

Comment:
`--amdgpu-target` is a `hipcc` only option.

It looks like the CMake logic in `hip-config.cmake` has some auto-detection and we need to combine it with our `AMReX_AMD_ARCH` control as in CUDA.

Reply:
```suggestion
      $<$<COMPILE_LANGUAGE:CXX>:-m64> )

  if(${_this_comp} STREQUAL hipcc)
      target_compile_options(amrex PUBLIC
          $<$<COMPILE_LANGUAGE:CXX>:-m64 --amdgpu-target=${AMReX_AMD_ARCH}> )
  endif()
```